<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>小工具-ffmpeg无损剪辑命令产生器</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,1">
    <item>
     <layout class="QVBoxLayout" name="vlayout_left" stretch="1,3">
      <item>
       <widget class="QGroupBox" name="grp_file">
        <property name="title">
         <string>选定文件</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="FileDropLabel" name="lbl_file_drop">
           <property name="acceptDrops">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>将待剪辑的文件拖到这里即可选定文件</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="grp_intervals">
        <property name="title">
         <string>时间段列表</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QGridLayout" name="glayout_intervals">
           <item row="4" column="1">
            <spacer name="vs_intervals_buts">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="0" column="1">
            <widget class="QToolButton" name="tbtn_move_up">
             <property name="text">
              <string>上移</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QToolButton" name="tbtn_remove">
             <property name="text">
              <string>删除</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QToolButton" name="tbtn_move_down">
             <property name="text">
              <string>下移</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QToolButton" name="tbtn_clear">
             <property name="text">
              <string>清空</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0" rowspan="5">
            <widget class="QTableView" name="tabv_intervals">
             <property name="selectionBehavior">
              <enum>QAbstractItemView::SelectRows</enum>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="hlayout_add_interval">
           <item>
            <layout class="QFormLayout" name="flayout_add_interval">
             <item row="0" column="0">
              <widget class="QLabel" name="lbl_interval_begin">
               <property name="text">
                <string>开始时间</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <layout class="QHBoxLayout" name="hlayout_add_interval_begin">
               <item>
                <widget class="QSpinBox" name="spin_interval_begin_hour">
                 <property name="maximum">
                  <number>999999999</number>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="lbl_colon_0">
                 <property name="text">
                  <string>:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSpinBox" name="spin_interval_begin_mins">
                 <property name="maximum">
                  <number>59</number>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="lbl_colon_1">
                 <property name="text">
                  <string>:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSpinBox" name="spin_interval_begin_secs">
                 <property name="maximum">
                  <number>59</number>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="lbl_interval_end">
               <property name="text">
                <string>截止时间</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <layout class="QHBoxLayout" name="hlayout_add_interval_end">
               <item>
                <widget class="QSpinBox" name="spin_interval_end_hour">
                 <property name="maximum">
                  <number>999999999</number>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="lbl_colon_2">
                 <property name="text">
                  <string>:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSpinBox" name="spin_interval_end_mins">
                 <property name="maximum">
                  <number>59</number>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="lbl_colon_3">
                 <property name="text">
                  <string>:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSpinBox" name="spin_interval_end_secs">
                 <property name="maximum">
                  <number>59</number>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QPushButton" name="pbtn_add_interval">
             <property name="text">
              <string>增加一个时间段</string>
             </property>
             <property name="shortcut">
              <string>Ctrl+Return</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="vlayout_right">
      <item>
       <widget class="QGroupBox" name="grp_source">
        <property name="title">
         <string>源码预览/导入源码</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="1" column="0">
          <spacer name="hs_parse_source">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>290</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="1" column="2">
          <widget class="QToolButton" name="tbtn_parse_source">
           <property name="text">
            <string>解析源码</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0" colspan="3">
          <widget class="QTextEdit" name="txtedt_source">
           <property name="lineWrapMode">
            <enum>QTextEdit::NoWrap</enum>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QCheckBox" name="chkbox_format_source">
           <property name="text">
            <string>格式化源码</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="grp_output">
        <property name="title">
         <string>输出命令</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="lbl_output_path">
             <property name="text">
              <string>视频输出路径</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineedt_output_path"/>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QTextBrowser" name="txtbrw_output">
           <property name="lineWrapMode">
            <enum>QTextEdit::NoWrap</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_T">
    <property name="title">
     <string>任务(&amp;T)</string>
    </property>
    <addaction name="act_new_task"/>
   </widget>
   <addaction name="menu_T"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="act_new_task">
   <property name="text">
    <string>新建任务</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FileDropLabel</class>
   <extends>QLabel</extends>
   <header>ui.FileDropLabel</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>tabv_intervals</tabstop>
  <tabstop>spin_interval_begin_hour</tabstop>
  <tabstop>spin_interval_begin_mins</tabstop>
  <tabstop>spin_interval_begin_secs</tabstop>
  <tabstop>spin_interval_end_hour</tabstop>
  <tabstop>spin_interval_end_mins</tabstop>
  <tabstop>spin_interval_end_secs</tabstop>
  <tabstop>pbtn_add_interval</tabstop>
  <tabstop>tbtn_move_up</tabstop>
  <tabstop>tbtn_move_down</tabstop>
  <tabstop>tbtn_remove</tabstop>
  <tabstop>txtedt_source</tabstop>
  <tabstop>txtbrw_output</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>spin_interval_begin_hour</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>on_spin_interval_begin_value_changed(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>117</x>
     <y>516</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spin_interval_begin_mins</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>on_spin_interval_begin_value_changed(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>196</x>
     <y>516</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spin_interval_begin_secs</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>on_spin_interval_begin_value_changed(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>254</x>
     <y>516</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spin_interval_end_hour</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>on_spin_interval_end_value_changed(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>117</x>
     <y>544</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spin_interval_end_mins</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>on_spin_interval_end_value_changed(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>196</x>
     <y>544</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spin_interval_end_secs</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>on_spin_interval_end_value_changed(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>254</x>
     <y>544</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>on_spin_interval_begin_value_changed(int)</slot>
  <slot>on_spin_interval_end_value_changed(int)</slot>
 </slots>
</ui>
